src/st/np3/np3.h

```diff
@@ -85,9 +85,11 @@ extern void func_801BB044(s16);
 extern void func_801BB140(s16);
 extern void func_801BB2F8(s16);
 extern void func_801BB3F4(s16);
-extern s32 UnkCollisionFunc(u16* hitSensors, s16 sensorCount);
+extern s32 UnkCollisionFunc(s16* hitSensors, s16 sensorCount);
 extern s32 GetPlayerCollisionWith(Entity* entity, u16 w, u16 h, u16 flags);
-extern s32 GetDistanceToPlayerX(void);
+#ifdef VERSION_PC
+extern s16 GetDistanceToPlayerX(void);
```

Comment:
Defining this header causes mismatches on PSX, I've left it in but conditional for PC only to avoid the warning there